object/project.go

```diff
@@ -0,0 +1,54 @@
+package object
+
+import (
+	"time"
+	"xorm.io/builder"
+)
+
+type Project struct {
+	Id        int64
```

Comment:
We should keep the first 3 fields the same for all objects:

https://github.com/open-ct/openitem/blob/de5bfc4185fb368bfe947d10ae8f721a2c8ca8f6/object/dataset.go#L18-L20

The id will be: `Owner/Name` together.